packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -393,16 +401,19 @@ class AppDomain extends Domain {
       _sendAppEvent(app, 'started');
     });
 
-    await app._runInZone(this, () async {
+    await app._runInZone<Null>(this, () async {
```

Comment:
`_runInZone<void>`?

Reply:
Hmm, at runtime, this seemed to want a `Null`.